app.py

```diff
@@ -180,6 +181,8 @@ def publish_request_iam_entitlement(context, req, body):
     if 'iamEntitlementRequestTopic' in req:
         sns_helper = SNSLibrary(context)
         req['credentials'] = body['credentials']
+        if req.get("trails"):
```

Comment:
instead of calling `trails`, we shoudl call it `loggingAccount`. i got confused while reading hte code, i was trying to understand what `trails` is.

Also, i don't think there will be more than 1.